### DISCUSSION TITLE
Potential PAC3 Compat?

### DISCUSSION THREAD

**Discussion by laptoppe** [VickyFrenzy/gmod-enhanced-camera-2#44] [Ideas]
I've played around with PAC3 for a bit, and figured that it's possible to modify the standard c_hands viewmodel by setting the `hands` enitity as the owner of a group on root level.

I was thinking it could be possible to do something similar with the first person model in this mod if it used a separate entity to show the playermodel, but I'm not versed well enough in gmod modding to be able to tell.

**Comment by VickyFrenzy** [comment]
If you are delving into all PAC3 could offer, you might not even need a camera addon

If I remember correctly it's possible to setup and tweak your first person camera however you like with PAC3
